server/index.js

```diff
@@ -154,29 +205,8 @@ app.get('/api/user/:userId/workouts', (req, res, next) => {
     .catch(err => next(err));
 });
 
-app.post('/api/sign-up', (req, res, next) => {
-  const { username, password } = req.body;
-  if (!username || !password) throw new ClientError(400, 'ERROR: Username and password are required fields.');
-  argon2
-    .hash(password)
-    .then(hashedPassword => {
-      const params = [username, hashedPassword];
-      const sql = `
-      insert into "users" ("username", "hashedPassword")
-      values              ($1, $2)
-      returning "userId", "username", "createdAt";
-      `;
-      return db.query(sql, params);
-    })
-    .then(result => {
-      const [user] = result.rows;
-      res.status(201).json(user);
-    })
-    .catch(err => next(err));
-});
-
-app.post('/api/new-workout', (req, res, next) => {
-  const userId = 1;
+app.post('/api/new-workout/user/:userId', (req, res, next) => {
+  const userId = Number(req.params.userId);
```

Comment:
You should take the user id from the `req.user` so that it's based on the token and not the url route.